website/docs/about/contribution.md

```diff
@@ -1,34 +1,110 @@
 ---
 title: Contribution
+description: How to collaborate with the Design System team and contribute to the system.
 navigation:
   order: 104
 ---
 
 !!! Info
 
-At this time we are not accepting external contributions from non-HashiCorp employees. Many resources and links on this page are only available to HashiCorp employees.
+We are currently only accepting collaborations from HashiCorp employees. Resources and links on this page may only be available to HashiCorp employees.
+
 !!!
 
-## Types of contribution
+Thank you for your interest in contributing to the Helios Design System. We think of contributions as an opportunity for **collaboration across teams** and are open to various forms of collaboration, even direct contribution, as long as they follow our [collaborative process](/about/contribution/#our-collaborative-process). 
+
+## Collaboration types
+
+Possible forms of collaboration include, but are not limited to: 
+
+- [Direct contributions](/about/contribution/#direct-contributions), like bug fixes, documentation improvements, small enhancements to existing artifacts, etc.
+- Net-new artifacts, like icons, components, patterns, etc.
+- Obtaining support, feedback, and usage guidance from the Design System team.
+
+## Our collaborative process
+
+Our collaborative process is intended to be a guideline, not a playbook. Each collaboration may be different depending on the complexity of the proposal. This process generally works best for larger collaboration projects, but we’ll help define the necessary steps based on your project type during the kickoff.
+
+### Submit a new request
+
+Before submitting a new request, check that the issue doesn’t already exist in our [backlog](https://hashicorp.atlassian.net/jira/software/c/projects/HDS/boards/1082). If no request exists, create a new issue using our [intake form](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/2648080407/PROTOTYPE+Helios+HDS+Support+Intake+Form). Creating an issue allows us to track and prioritize all requests in one central location easily.
```

Comment:
I need to think of a better suggestion, but this link is only partially complete from a backlog perspective as it wouldn't include things we've prioritize independent from consumers requests (which may overlap)

Reply:
@Dhaulagiri Is there a place in Jira where **all** backlog tickets are stored?

Reply:
Not in the sense that we're looking for here. Most likely the [rollout tracker](https://go.hashi.co/hds-rollout) is still the best we have as it _should_ contain a representative list of all the potential projects we're aware of, but the introduction of Feature Requests is complicating that some (I need to investigate that more still).

Reply:
The rollout tracker only shows issues connected to epics; not all consumer-requested issues (or even issues we add for fixes/improvements) are connected to epics. My goal with asking consumers to look at our backlog before submitting an issue is to avoid duplicate issues; instead, I'd like them to add comments/upvote the existing issue if it's already there. Consumer requests may be bug fixes/issues, not complex projects or new components/patterns, so I'm not sure the rollout tracker is representative enough.

Reply:
That's a good/fair point. We can alway start with what you have and if we come up with something better we can update it.